applications/derive_mirror_rnda.py

```diff
@@ -210,6 +230,7 @@ def plotMeasuredDistribution(file, **kwargs):
     )
 
     args = parser.parse_args()
+    containment_fraction_percent = int(args.containment_fraction*100)
```

Comment:
You need to make sure the containment radius is actually in the right range for this (and for other things).

I suggest you use the built in functionality of argparse and simply add `choices=range(0,1)` to the that input. You can even add then `metavar="[0-1]"` and then my comment above about the interval might not be necessary.

Reply:
I've added now a method to the type definition for containment_fraction to check for a value inside the [0., 1.] interval.

As discussed, I will add a wrapper for the argparser where this can be included for general use.